.github/workflows/tox.yml

```diff
@@ -21,5 +21,6 @@ jobs:
       run: |
         python -m pip install --upgrade pip setuptools wheel
         pip install tox tox-gh-actions
+        python3 -m pip install -e ".[dev]"
```

Comment:
This should not be required. That's what tox should be doing

Reply:
Fixed